src/control/common/system_rpc.go

```diff
@@ -0,0 +1,100 @@
+//
+// (C) Copyright 2019 Intel Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+// The Government's rights to use, modify, reproduce, release, perform, display,
+// or disclose this software are subject to the terms of the Apache License as
+// provided in Contract No. 8F-30005.
+// Any reproduction of computer software, computer software documentation, or
+// portions thereof marked with this legend must also reproduce the markings.
+//
+
+package common
+
+import (
+	"errors"
+	"net"
+
+	mgmtpb "github.com/daos-stack/daos/src/control/common/proto/mgmt"
+	"github.com/daos-stack/daos/src/control/logging"
+)
+
+// MembersToPB converts internal member structs to protobuf equivalents.
+func MembersToPB(members SystemMembers) []*mgmtpb.SystemMember {
```

Comment:
Wondering if this conversion logic and SystemMember related stuff really belongs in common. Not something to change right now, but something to think about in the future. Feels sort of similar to security, where the domain is specific but used by both client and server.

Reply:
do you think it needs its own package?

Reply:
That's what I was thinking. Or some package including only content related to these client/server communications. But I'm not sure if there's enough content to justify breaking it out, we'd need to look into it.

Reply:
may well be warranted in the future